doc/release-notes.md

```diff
@@ -102,6 +102,22 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
+- The `getpeerinfo` RPC no longer returns the `inbound` or `addnode` fields by
+  default. These two fields will be fully removed in the next major release.
+  They can be accessed with the configuration options
+  `-deprecatedrpc=getpeerinfo_inbound` and `-deprecatedrpc=getpeerinfo_addnode`
+  respectively. However, it is recommended to instead use the `connection_type`
+  field, which will indicate whether the peer is inbound, addnode(manual), or
```

Comment:
```suggestion
  field.
```

Probably can remove the extended documentation

Reply:
done